### PR TITLE
Review Schedule section

### PR DESCRIPTION
**Pull request by Sergiodero**
This PR reviews working links for all pages in the schedule section under Technical Documentation. This includes the following pages:

- [x] Reference
- [x] Best Practices
- [x] Data examples - Attributions
- [x] Data examples - Continuous stops
- [x] Data examples - Fares v1 (_Note: no changes were made to this page_)
- [x] Data examples - Fares v2
- [x] Data examples - Feed Info (_Note: no changes were made to this page_)
- [x] Data examples - Flexible Services
- [x] Data examples - Frequencies (_Note: no changes were made to this page_)
- [x] Data examples - Pathways
- [x] Data examples - Routes, Stops & trips (_Note: no changes were made to this page_)
- [x] Data examples - Text-to-speech (_Note: no changes were made to this page_)
- [x] Data examples - Transfers (_Note: no changes were made to this page_)
- [x] Data examples - Translations
- [x] Change history - Recent additions (_Note: no changes were made to this page_)
- [x] Change history - Full revision history (_Note: no changes were made to this page_)

This PR will be updated as the review progresses.

### CODE REVIEW

**Comment by Sergiodero** [comment]
I managed to finish the review for the schedule section, I only have a couple of pre-review comments below:

- I had to remove some sources for some example pages since the external references we're no longer valid (see [Continuous Stops- Pickup and drop off everywhere](https://staging.gtfs.org/documentation/schedule/examples/continuous-stops/#pickup-and-drop-off-everywhere) example source). This will leave some examples with sources and other without. I think is is generally okay but I want to get your thoughts.
- While reviewing I noticed that the [flex example page is currently listed as Flexible services](https://staging.gtfs.org/documentation/schedule/examples/flex/) in the navigation panel and its in-page title is Demand Responsive Services. I personally think that Flexible services might be more intuitive, but I also know that this might be inconsistent with the current name in gtfs.org. Do you prefer to harmonize everything to "Flexible Services" or "Demand Responsive Services"?
- There are several invalid external references in the best practices file (e.g. second example for [lasso routes](https://staging.gtfs.org/documentation/schedule/schedule_best_practices/#lasso-routes)), I'm leaving them as they are as I think this is something that should be changed in the google/transit repo rather than here. I'm not thinking of going into that at this time, but lmk if you think we should do otherwise.

Let me know what you think about these points and please go ahead with the review whoever have capacity @tzujenchanmbd @isabelle-dr Thanks again!

File: docs/documentation/schedule/examples/fares-v2.md
Comment (tzujenchanmbd) [inline docs/documentation/schedule/examples/fares-v2.md:1]: There are 4 videos in the current gtfs.org fares v2 example pages. Have we deleted them or moved them to other places on purpose?

**Review by tzujenchanmbd — COMMENTED** [review]
@Sergiodero Thanks for working on this! The changes LGTM!
Just have 1 question regarding fares v2 videos and I also directly pushed 2 commits in the same branch, please help review those changes as well.

**Review by tzujenchanmbd — APPROVED** [review]

**Comment by isabelle-dr** [comment]
Given that @tzujenchanmbd reviewed, I won't review this PR. Thanks to you both!

File: docs/documentation/schedule/examples/fares-v2.md
Comment (Sergiodero) [reply docs/documentation/schedule/examples/fares-v2.md:1]: Thanks for catching this! Our version of the Fares v2 example page is outdated, I'll work on updating it.

**Comment by Sergiodero** [comment]
Thanks @tzujenchanmbd, I'll work on updating the Fares v2 example page and review your commits, any thoughts on [the points that I shared in my last comment](https://github.com/MobilityData/elrond.gtfs.org/pull/75#issuecomment-2148451638)? @isabelle-dr feel free to share your thoughts as well, no need to review the whole PR

**Comment by tzujenchanmbd** [comment]
Re: Demand Responsive Services vs Flexible Services
Slightly inclined to demand responsive on the example page because it seems like a name being used more in transportation industry. We use flexible services in features but it looks ok to me as well because it also includes continuous stops.

The other 2 points lgtm

**Comment by Sergiodero** [comment]
Thanks @tzujenchanmbd I added two final commits, one to add the videos in the Fares v2 example page and another one to update the flex example page name. Merging now!

**Pull request merged**